src/hidimstat/statistical_tools/multiple_testing.py

```diff
@@ -0,0 +1,188 @@
+import numpy as np
+
+
+def cal_fdp_power(selected, non_zero_index):
+    """
+    Calculate power and False Discovery Proportion
+
+    Parameters
+    ----------
+    selected: list index (in R format) of selected non-null variables
```

Comment:
Is the function meant to be part of the public API ? (I think it shouldn't) If yes, the docstring has to be fixed.